res/TensorFlowPythonExamples/examples/non_max_suppression_with_scores_2/__init__.py

```diff
@@ -0,0 +1,13 @@
+import tensorflow as tf
+
+max_output_size = tf.compat.v1.constant(6)
+iou_threshold = tf.compat.v1.constant(0.5)
+score_threshold = tf.compat.v1.constant(0.6)
+soft_nms_sigma = tf.compat.v1.constant(0.5)
+
+in_boxes_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(12, 4), name="Hole")
+in_scores_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(12), name="Hole")
+
+non_max_suppression_with_scores_ = tf.compat.v1.image.non_max_suppression_with_scores(
```

Comment:
> AttributeError: module 'tensorflow._api.v1.compat.v1.image' has no attribute 'non_max_suppression_with_scores'

ditto

Reply:
Done, added line comment about TF version requirement for `non_max_suppression_with_scores`.